packages/gasket-plugin-log/README.md

```diff
@@ -0,0 +1,173 @@
+# @gasket/plugin-log
```

Comment:
Whoa whoa - this plugin should be gone and replaced with `@gasket/plugin-logger`